.github/workflows/analyze.yml

```diff
@@ -0,0 +1,33 @@
+name: "Analyze"
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+    paths-ignore:
+      - "**/*.md"
+
+jobs:
```

Comment:
github supports this ootb now, so you could switch to that in theory https://github.com/getsentry/sentry-android-gradle-plugin/blob/528f265c16a9539c40f4f278145493c65ba95620/.github/workflows/build.yml#L10-L12C27